portal/static/js/profile.js

```diff
@@ -1490,6 +1490,12 @@
                 var newValue = event.target.value;
                 this.orgsSelector.selectedState = newValue;
             },
+            isAcceptOnNextOrg: function(orgName) {
```

Comment:
Would it make sense to keep this org-specific config in the org API/JSON?

Reply:
hmm, that is a good idea actually.  The config variable I am referencing here was initially created just so the backend can identify the Accept terms on next work flow (e.g. MUSIC).  I am referencing it at the frontend for consistency.